ngx-tools/src/update-control-on-input-changes/update-control-on-input-changes.spec.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  SimpleChanges,
+  SimpleChange,
+} from '@angular/core';
+
+import { updateControlOnInputChanges } from './update-control-on-input-changes';
+import { FormControl } from '@angular/forms';
+
+
+describe(`updateControlOnInputChanges`, () => {
+  const changed = {
+    item1: new SimpleChange(undefined, true, true),
+    item2: new SimpleChange('foo', 'foo', false),
+  } as SimpleChanges;
+  const control: FormControl = new FormControl();
+
+
+  test(`should return false if the changes object or key are missing`, () => {
+    expect(updateControlOnInputChanges(null, 'foo', control)).toEqual(false);
```

Comment:
Interesting... I would have expected you to have to actually test sending in a null key and null form control to fully test this first `if` statement.. but coverage is 100%. I may be over-testing a few places..